### DISCUSSION TITLE
Learning JS

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#696] [(D) Developer]
https://javascript.info/

https://developer.mozilla.org/en-US/docs/Web/JavaScript

https://github.com/leonardomso/33-js-concepts

**Comment by ArcerionDev** [comment]
good now people will stop opening issues asking how to code

**Reply by rusprice** [reply]
*learns hello world in JS*
"can I be a staff member I know js!!!"
